sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java

```diff
@@ -258,10 +283,13 @@ public Mono<Response<BinaryData>> getCompletionsWithResponse(
      *     text that continues from or "completes" provided prompt data along with {@link Response} on successful
      *     completion of {@link Mono}.
      */
-    @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Response<BinaryData>> getChatCompletionsWithResponse(
             String deploymentId, BinaryData chatCompletionsOptions, RequestOptions requestOptions) {
+        if (this.openAIServiceClient != null) {
+            return this.openAIServiceClient.getChatCompletionsWithResponseAsync(
+                    deploymentId, chatCompletionsOptions, requestOptions);
+        }
```

Comment:
Should streaming methods also be updated?

Reply:
Streaming methods call non-Stream xxxWithResponse methods internally. So they are covered.

Reply:
How does the streaming methods know which endpoint to call?

Reply:
https://github.com/Azure/azure-sdk-for-java/blob/feature/open-ai/sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java#L364

Stream API calls `getCompletionsWithResponse` which returns the result to the Stream API. But it could have an issue if the OpenAI and Azure OpenAI data structures are different.